frontend/app/admin/src/views/settings/organization-records/EditOrganizationRecordSettings.vue

```diff
@@ -0,0 +1,73 @@
+<template>
```

Comment:
Probeer in de mate van het mogelijk een .vue component altijd duidelijk te maken wat het is. Nu heet het EditOrganizationRecordSettings, maar het is niet duidelijk of het een volledige view is. In dit geval is 'box' de beste benaming consistent met de andere benamingen. Dus best renamen naar `EditOrganizationRecordSettingsBox`